imap_processing/tests/spice/test_pointing_frame_handler.py

```diff
@@ -0,0 +1,177 @@
+"""Tests Pointing Frame Generation."""
+
+import shutil
+import sys
+from pathlib import Path
+
+import matplotlib.pyplot as plt
```

Comment:
```suggestion
```
We don't want matplotlib as any dependency in my opinion. It can be an optional one if we want, but we shouldn't bring it in as a top-level import.

Reply:
What about this?
[tool.poetry.extras]
dev = ["pre-commit", "ruff", "mypy"]
doc = ["numpydoc", "pydata-sphinx-theme", "sphinx", "sphinxcontrib-openapi"]
test = ["openpyxl", "pytest", "pytest-cov"]
tools= ["openpyxl", "pandas"]
plotting = ["matplotlib"]